ISISComputingGroup/BeckhoffTestRunner
# Update to use the AutomationTools to set up test axes

See https://github.com/ISISComputingGroup/IBEX/issues/6637

Now uses https://github.com/Simon-Cooper/twinCATAutomationTools to create a test axis, meaning we can test against the collaboration master branch

To test:
* Run `build.bat`
* Confirm it runs without errors
* Open the PLC solution file and confirm that it contains two test axes
